src/containers/SettingsEditor/SettingsEditor.styled.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components'
+
+type Props = {
+  currentSelection: string
```

Comment:
This styled prop should use $ prefix `$currentSelection`.

https://styled-components.com/docs/api#transient-props